net-libs/tox/metadata.xml

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+	<maintainer type="person">
+		<email>JoMull01@protonmail.com</email>
+		<name>Josiah Mullins</name>
+	</maintainer>
 	<use>
```

Comment:
You need someone to commit for you.

Reply:
Alright, how do I proceed?

Reply:
If you'd like to work through proxy-maint, https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide. Otherwise, you need to find a specific developer and talk to him.